doc/src/reference.xml

```diff
@@ -561,13 +561,15 @@ path-constant DATA : data/a.txt ;
       <varlistentry>
         <term><literal>cflags</literal></term>
         <term><literal>cxxflags</literal></term>
+        <term><literal>compileflags</literal></term>
```

Comment:
I will remove the change but I strongly feel that the documentation, even if correct in general, will just serve to lead end-users of Boost Build astray. At the very least a further explanation should be added directing the end-user to look at the toolset for his compiler where the meaning of 'cflag' often will be different.